### PR TITLE
Link to build-states

### PR DESCRIPTION
**Pull request by pbuckley**
Hoping to disambiguate scheduled jobs vs scheduled_builds, I get them mixed in my mind.

### CODE REVIEW

**Comment by ozdenbuildkite** [comment]
@pbuckley updated some of the wording, let me know what you think!

File: pages/pipelines/build_timeouts.md [hunk 1]
```diff
@@ -31,8 +31,11 @@ Note that command step timeouts don't apply to trigger steps and block steps.
 
 ## Scheduled job expiry
 
-Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled.
+Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled.
```
Comment (pbuckley) [inline pages/pipelines/build_timeouts.md:34]:
```suggestion
Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled.
```
Just dropping a trailing space after the period here, likely left from my initial change.

File: pages/pipelines/build_timeouts.md [hunk 2]
```diff
@@ -36,3 +36,6 @@ Scheduled job expiry helps you avoid having lingering jobs that are never assign
 By default, jobs are canceled when not picked up for 30 days. This will cause the corresponding build to fail.
 
 You can override the default by setting a shorter value in your organization's [Pipeline Settings page](https://buildkite.com/organizations/~/pipeline-settings).
+
+Scheduled job limits should not be confused with [scheduled builds](/docs/pipelines/scheduled-builds), a scheduled build's jobs will still go through the [build states](/docs/pipelines/defining-steps#build-states) and the timeout will apply once its individual jobs are in scheduled state waiting for agents.
```
Comment (mbelton-buildkite) [inline pages/pipelines/build_timeouts.md:40]: Minor tweaks:

```suggestion
Scheduled job limits should not be confused with [scheduled builds](/docs/pipelines/scheduled-builds). A scheduled build's jobs will still go through the [build states](/docs/pipelines/defining-steps#build-states), and the timeout will apply once its individual jobs are in the scheduled state waiting for agents.
```

**Review by mbelton-buildkite — APPROVED** [review]
One suggestion but otherwise looks good :)

**Review by gilesgas — APPROVED** [review]

**Pull request merged**